### ISSUE TITLE
Sala de espera

### ISSUE THREAD

**Issue by narispe** [IIC2233/Syllabus#300]
* [X] Leí las reglas del foro (https://github.com/IIC2233/syllabus/issues/1)
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues)
* [X] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía (https://github.com/IIC2233/syllabus/issues/293)
* [X] Mi duda no se trata sobre una librería, *built-in* o mala práctica, ya que eso se pregunta en la *issue* creada para ello.
* [X] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al Jefe de Bienestar o al correo del curso (https://iic2233.github.io/contacto/)
* [X] Utilizaré un título descriptivo y llenaré correctamente esta plantilla
* [X] De ser necesario, solo colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado copia

### Duda

Hola, en la sección 4.1 se menciona una "Sala de espera", esta corresponde a la "Sala principal"?

**Comment by Emiliax16** [comment]
Hola @narispe . Toda la razón. Perdón por el error de tipeo. En vez de "Sala de espera" en la sección 4.1, segundo párrafo, debería decir "Sala principal".

Dentro del enunciado se habla siempre de una Sala principal, por lo que no hay ninguna "Sala de espera" extra ni nada. Gracias por el aviso!

**Comment by ChristianCarrillo07** [comment]
Me tomo de esta Issue para consultar:
En punto 8 (Avance de tarea) se habla de un servidor "capaz de recibir nombres para ingresar a la **sala de espera**".
Esta sala de espera tambien sería un error?

**Comment by Emiliax16** [comment]
Exacto, nuevamente perdón por el error de tipeo😭 y muchas gracias por notificarlo 💯
